wpilibc/src/main/native/cpp/PIDController.cpp

```diff
@@ -498,8 +520,8 @@ void PIDController::SetToleranceBuffer(int bufLength) {
  * This will return false until at least one input value has been computed.
  */
 bool PIDController::OnTarget() const {
+  if (StatusIsFatal()) return false;
```

Comment:
I left a newline before the lock_guard to contextually separate it from the GetError() call. I would add a newline after the if statement and before the lock_guard.

Reply:
Ok.

Reply:
To be clear, I meant one after the if statement but before GetError() and one after GetError() but before the lock_guard.

Reply:
Alright, should be good now.